### DISCUSSION TITLE
Daybreak(unblock Crosh /Terminal)

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1229] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I cant install secure shell extension . 
Is there a website I can do the exploit on like ssh emulator or something??/

**Comment by Luigi-Master** [comment]
do you have an Android phone

**Reply by ghost** [reply]
no

**Reply by Luigi-Master** [reply]
sorry I'll try to find new exploits, I'm testing some right now actually

**Reply by ghost** [reply]
ok thanks

**Comment by akabutnicer** [comment, marked as answer]
> I cant install secure shell extension . Is there a website I can do the exploit on like ssh emulator or something??/

If there was it wouldn't matter. An ssh emulator cannot access your Chromebook.
